src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -361,14 +362,58 @@ invalidated when a device disconnects. This means your code should always
 retrieve (through `getPrimaryService(s)`, `getCharacteristic(s)`, etc.) these
 attributes after reconnecting.
 
+### Read and write to Bluetooth Descriptors
+
+Bluetooth GATT descriptors are defined attributes that describe a
+characteristic value. You can access them, read and write to in a similar way
+to Bluetooth GATT characteristics.
+
+Let's see for instance how to read the user description of the measurement
+interval of the device's health thermometer.
+
+In the example below, `health_thermometer` is the [Health
+Thermometer Service](https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.service.health_thermometer.xml), `measurement_interval` the [Measurement Interval Characteristic](https://developer.bluetooth.org/gatt/characteristics/Pages/CharacteristicViewer.aspx?u=org.bluetooth.characteristic.measurement_interval.xml), and `gatt.characteristic_user_description` the [Characteristic User Description Descriptor](https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.descriptor.gatt.characteristic_user_description.xml).
+
```

Comment:
They are capitalized as you can see https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.descriptor.gatt.characteristic_user_description.xml but not `descriptor`.
I've uncapitalized `service`, `characteristic`, `descriptor`.
Done.